lib/services/devices/devices-NGSI-v2.js

```diff
@@ -432,7 +343,7 @@ function updateRegisterDeviceNgsi2(deviceObj, entityInfoUpdated, callback) {
                     deviceObj.subservice
                 ),
                 apply(extractDeviceDifference, deviceObj),
-                createInitialEntityNgsi2,
+                createInitialEntityNgsi2Fake,
```

Comment:
Is this really needed? Why don't just remove this method from the waterfall ?

Reply:
* This methods makes a bypass in updateRegisterDeviceNgsi2 to allow not change
 * extractDeviceDifference and combineWithNewDevice methods

Reply:
NTC